memcopy.h

```diff
@@ -22,6 +22,12 @@ typedef uint8x16_t inffast_chunk_t;
 #  define INFFAST_CHUNKSIZE sizeof(inffast_chunk_t)
 #endif
 
+# if defined(X86_SSE2)
```

Comment:
May need to fix whitespacing here. # if contains a space between the two. And also at endif.

Reply:
It does look like correct indentation for a nested if is used from this line and down, but the two if blocks just above here break the pattern and make it hard to understand, they should also preferably be indented so we can easily see where this block starts (at MEMCOPY_H in line 4).

Reply:
Honestly, I personally prefer to indent the # sign as well, instead of leaving it at the beginning of the line. That makes the code easier to read at a glance. I use syntax highlighting, so I still see the preprocessor macros are there.